setup-runner.sh

```
@@ -0,0 +1,102 @@
+
+# Create actions-runner directory if it doesn't exist
+if [ ! -d "actions-runner" ]; then
+    mkdir actions-runner
+    echo "📁 Created actions-runner directory"
+fi
+
+cd actions-runner
+
+# Download the latest runner package (Linux x64)
+echo "📦 Downloading GitHub Actions Runner..."
+RUNNER_VERSION=$(gh api repos/actions/runner/releases/latest --jq '.tag_name' | sed 's/^v//')
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)